Agreements/External/D_BUlow.txt

```diff
@@ -1,8 +1,7 @@
-FIR_COPX:*:*:KONAN:EBAW:*:London AC Dover:EBBU low:*:23000:VABIK
-FIR_COPX:*:*:KONAN:EBBR:*:London AC Dover:EBBU low:*:23000:VABIK
```

Comment:
Seem to have lost one of the agreements here

Reply:
Just noticed that, readding it now